cli/info.go

```diff
@@ -56,60 +56,60 @@ func (v *InfoCommand) runInfo() error {
 }
 
 func prettyPrintInfo(cli *Cli, info *types.SystemInfo) error {
-	fmt.Fprintln(os.Stdout, "Containers: ", info.Containers)
-	fmt.Fprintln(os.Stdout, " Running: ", info.ContainersRunning)
-	fmt.Fprintln(os.Stdout, " Paused: ", info.ContainersPaused)
-	fmt.Fprintln(os.Stdout, " Stopped: ", info.ContainersStopped)
-	fmt.Fprintln(os.Stdout, "Images: ", info.Images)
-	fmt.Fprintln(os.Stdout, "ID: ", info.ID)
-	fmt.Fprintln(os.Stdout, "Name: ", info.Name)
-	fmt.Fprintln(os.Stdout, "Server Version: ", info.ServerVersion)
-	fmt.Fprintln(os.Stdout, "Storage Driver: ", info.Driver)
-	fmt.Fprintln(os.Stdout, "Driver Status: ", info.DriverStatus)
-	fmt.Fprintln(os.Stdout, "Logging Driver: ", info.LoggingDriver)
-	fmt.Fprintln(os.Stdout, "Volume Drivers: ", info.VolumeDrivers)
-	fmt.Fprintln(os.Stdout, "Cgroup Driver: ", info.CgroupDriver)
-	fmt.Fprintln(os.Stdout, "Default Runtime: ", info.DefaultRuntime)
+	fmt.Fprintf(os.Stdout, "Containers: %d\n", info.Containers)
+	fmt.Fprintf(os.Stdout, " Running: %d\n", info.ContainersRunning)
+	fmt.Fprintf(os.Stdout, " Paused: %d\n", info.ContainersPaused)
+	fmt.Fprintf(os.Stdout, " Stopped: %d\n", info.ContainersStopped)
+	fmt.Fprintf(os.Stdout, "Images: %d\n", info.Images)
+	fmt.Fprintf(os.Stdout, "ID: %s\n", info.ID)
+	fmt.Fprintf(os.Stdout, "Name: %s\n", info.Name)
+	fmt.Fprintf(os.Stdout, "Server Version: %s\n", info.ServerVersion)
+	fmt.Fprintf(os.Stdout, "Storage Driver: %s\n", info.Driver)
+	fmt.Fprintf(os.Stdout, "Driver Status: %v\n", info.DriverStatus)
```

Comment:
Would it be better to print a list of items one by one?

Reply:
Well, I think we can use loop to print each item in list. However, I don't know how to use the field by integration side. We need to know the usage before changing the behavior.